docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -0,0 +1,339 @@
+---
+title: "Using asset checks to check data freshness | Dagster Docs"
+description: "Use freshness checks to identify data assets that are overdue for an update."
+---
+
+# Using asset checks to check data freshness
+
+A freshness check is a type of [asset check](/concepts/assets/asset-checks) that allows you to define the amount of time between materializations/observations for Dagster [assets](/concepts/assets/software-defined-assets).
```

Comment:
"define the amount of time" makes it sound like all freshness checks are based on a fixed time interval between updates, but some of the ways we allow defining freshness checks are a little different. I would lean towards just "A freshness check is a type of asset check that allows you to identify assets that are overdue for an update."

Reply:
For my own understanding, you're referring to the time anomaly option here, right? Since that doesn't really allow you to define an amount of time for freshness, as it does it for you based on past data.

Reply:
I'm referring to that as well as the `freshness_cron` parameter for hardcoded freshness. When you use that parameter, instead of defining the amount of time between updates, you essentially define a deadline for when an update is expected by.

Reply:
Missed that, ty! Updated the copy here.